home/templates/home/docs/applicant_guide.html

```diff
@@ -661,6 +661,8 @@ <h2 id='final-application'><a href='#final-application'>Final application</a></h
 		Carefully review the project description to familiarize yourself with the provided tasks and milestones. Once you have a clear understanding of the project, create a timeline outlining the tasks you plan to work on during the internship. Some interns have found it helpful to break down their project tasks by weeks.
 		Be prepared to adjust your timeline as needed to accommodate unexpected developments or changes in project scope. Make sure to take into account any time commitments you have during the Outreachy internship round.
 		If you are still working on your contributions and need more time, you can leave this blank and edit your application later.
+        In addition to your project timeline, plan to incorporate milestones where you share drafts of your work with your mentors and the community.
+        These milestones should occur at least every two weeks. This will not only help you stay on track but also help you get valuable feedback about your project.
```

Comment:
I wonder if we should tell them to incorporate that into their bi-weekly blog posts. We may want to revisit our blog prompts.

Reply:
I agree, but the blog prompts documentation would have to be updated first so it contains more detail about adding milestones, and we can link to that doc when we update this.